docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -130,37 +190,195 @@ spec:
     See [Create secret text](../implementing/tasks/#create-secret-text)
     for details.
 
+## Yaml Synopsis for container-runtime
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v?alpha?
+kind: KeptnTaskDefinition
+metadata:
+  name: <task-name>
+spec:
+  container
+    name: <container-name>
+    image: <image-name>
+    <other fields>
+```
+
+### Spec used only for container-runtime
+
+The `container-runtime` can be used to specify
+your own container image and define almost task you want to do.
+If you are migrating from Keptn v1,
+you can use a `container-runtime` to execute
+almost anything that you implemented with JES for Keptn v1.
+
+* **spec**
+  * **container** -- Container definition.
+    * **name** -- Name of the container that will run,
+      which is not the same as the `metadata.name` field
+      that is used in the `KeptnApp` resource.
+    * **image** -- name of the image you defined according to
+      [image reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#image)
+      and
+      [image concepts](https://kubernetes.io/docs/concepts/containers/images/)
+      and pushed to a registry
+    * **other fields** -- The full list of valid fields is available at
+      [ContainerSpec](../crd-ref/lifecycle/v1alpha3/#containerspec),
+      with additional information in the Kubernetes
+      [Container](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)
+      spec documentation.
+
+      TODO: Should we only link to the ContainerSpec here or should
```

Comment:
Here we can refer to the official kubernetes documentation about the containers, there is clearly stated what is required and what not

Reply:
done